tests/rest_api_tests.py

```diff
@@ -1,7 +1,9 @@
 """Unit Tests for Rest API."""
 
+import sys
 import unittest
-import rest_api
+sys.path.append('../')   # allows for importing of files not in the same directory | pylint: disable=line-too-long
```

Comment:
do that in the __init__.py. @cabreraem has a fix in another PR.

Reply:
That's the plan, just waiting for it to be approved